index.bs

```diff
@@ -2445,6 +2455,10 @@ optionally evidence of [=user consent=] to a specific transaction.
     :   <dfn>signature</dfn>
     ::  This attribute contains the raw signature returned from the authenticator. See [[#sctn-op-get-assertion]].
 
+    :   <dfn>transport</dfn>
+    ::  This attribute contains the transport used to communicate to the [=authenticator=], or `null` if the user agent does not have any transport information.
+        See [[#sctn-op-get-assertion]].
```

Comment:
Why have this reference to section _6.3.3. The authenticatorGetAssertion Operation_ as part of the description for `transport` when the value is the responsibility of the user agent and not the authenticator?

Perhaps just remove this reference?

Reply:
I also really feel there should be some description of the intended use of this new field. When I was first developing an RP implementation, my focus was on section 7, and for assertion ceremonies 7.2. Perhaps here in 5.2.2 is the right place for an explanatory note, or perhaps in section 7.2, either way there should be an indication that this value is a hint to the RP as to how the client communicated with the authenticator, for possible consideration in what might happen next in a user experience.